### DISCUSSION TITLE
求问：v2版本的微信支付，transfer账户转账功能

### DISCUSSION THREAD

**Discussion by kagamiii** [yansongda/pay#718] [General]
求问作者和万能的网友，微信这个transfer转账功能，用的【商家转账到零钱】还是【企业付款到零钱】？微信商户收到通知，要取消企业付款到零钱，以后全都用商家转账到零钱。

**Comment by yansongda** [comment]
v2 应该用的是  企业付款到零钱，具体可看文档里的链接

**Reply by kagamiii** [reply]
感谢 ^_^
